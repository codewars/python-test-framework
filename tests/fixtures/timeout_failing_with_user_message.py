import codewars_test as test


@test.describe("group 1")
def group_1():
    @test.timeout(0.01, "nope...")
    def test_1():
        while True:
            pass
        test.pass_()
