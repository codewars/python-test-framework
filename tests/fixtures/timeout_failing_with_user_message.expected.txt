
<DESCRIBE::>group 1

<FAILED::>Exceeded time limit of 0.010 seconds: nope...

<COMPLETEDIN::>30.41
